### PR TITLE
pythonPackages.pdfx: init at 1.3.1

### PR DESCRIPTION
**Pull request by marsam**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [X] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/pdfx/default.nix
Comment (dotlambda) [inline pkgs/development/python-modules/pdfx/default.nix]: I think you should package pdfminer2 instead of using an old version.

File: pkgs/development/python-modules/pdfx/default.nix
Comment (dotlambda) [inline pkgs/development/python-modules/pdfx/default.nix]: I have opened https://github.com/metachris/pdfx/pull/29 so that `fetchPypi` can be used.

**Review by dotlambda — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/pdfx/default.nix
Comment (dotlambda) [reply pkgs/development/python-modules/pdfx/default.nix]: Wasn't `pdfminer.six` explicitly replaced in https://github.com/metachris/pdfx/commit/cee76e10520ba5a36386dbf5eb43c8d9f3243e59#diff-2eeaed663bd0d25b7e608891384b7298?

Comment (marsam) [reply pkgs/development/python-modules/pdfx/default.nix]: > Wasn't pdfminer.six explicitly replaced in metachris/pdfx@cee76e1?

I didn't knew that, but seems that there is no significant changes in pdfminer2 https://github.com/pdfminer/pdfminer.six/compare/master...metachris:master.
pdfx works with pdfminer.six, but i can package pdfminer2 if you prefer it

Comment (dotlambda) [reply pkgs/development/python-modules/pdfx/default.nix]: Let's first wait for the maintainers answer to https://github.com/metachris/pdfx/pull/28.

Comment (dotlambda) [reply pkgs/development/python-modules/pdfx/default.nix]: As the package seems unmaintained, I'd say we just go with your change. However, please add a link to your PR as a comment.

File: pkgs/development/python-modules/pdfx/default.nix
Comment (dotlambda) [inline pkgs/development/python-modules/pdfx/default.nix]: no full stop: https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes

Comment (marsam) [reply pkgs/development/python-modules/pdfx/default.nix]: fixed :)

**Comment by dotlambda** [comment]
@GrahamcOfBorg build python2.pkgs.pdfx python3.pkgs.pdfx

**Pull request merged**
